docs/en/Configuration-Settings.md

```diff
@@ -8,31 +8,75 @@ priority: 1
+# Configuration in Alluxio
 
-# Configuration properties
+Alluxio runtime respects two sources of configuration settings:
 
-On startup, Alluxio loads the default (and optionally a site specific) configuration properties file
-to set the configuration properties.
+1. [Environment variables](#Environment-variables). This is a convenient way for beginner users or shell scripts to set
+a few basic properties when running Alluxio.
+2. [Configuration properties](#configuration-properties). This provides a way to customize any
+[supported Alluxio configure properties](#appendix).
 
-1. The default values of configuration properties of Alluxio are defined in
-`alluxio-default.properties`. This file can be found in Alluxio source tree and is typically
-distributed with Alluxio binaries. We do not recommend beginner users to edit this file directly.
+## Environment variables
 
-2. Each site deployment and application client can also override the default property values via
-`alluxio-site.properties` file. Note that, this file **must be in the classpath** of the Java VM in
-which Alluxio is running. The easiest way is to copy the site properties template in directory
+There are a few basic but very frequently used Alluxio configuration properties that can be set via the
+following environment variables:
+
+<table class="table table-striped">
+<tr><th>Environment Variable</th><th>Meaning</th></tr>
+<tr><td>`$ALLUXIO_MASTER_HOSTNAME`</td><td>hostname of Alluxio master, default to localhost.</td></tr>
+<tr><td>`$ALLUXIO_MASTER_ADDRESS`</td><td>Same as `ALLUXIO_MASTER_HOSTNAME`, deprecated since version 1.1 and
+will be remove in version 2.0.</td></tr>
+<tr><td>`$ALLUXIO_UNDERFS_ADDRESS`</td><td>under storage system address, default to
+`${ALLUXIO_HOME}/underFSStorage` which is a local file system.</td></tr>
+<tr><td>`$ALLUXIO_RAM_FOLDER`</td><td>The directory where a worker stores in-memory data, default to /mnt/ramdisk.</td></tr>
+<tr><td>`$ALLUXIO_JAVA_OPTS`</td><td>Java VM options for both Master and Worker.</td></tr>
+<tr><td>`$ALLUXIO_MASTER_JAVA_OPTS`</td><td>additional Java VM options for Master configuration.</td></tr>
+<tr><td>`$ALLUXIO_WORKER_JAVA_OPTS`</td><td>additional Java VM options for Worker configuration. Note that, by
+default `ALLUXIO_JAVA_OPTS` is included in both `ALLUXIO_MASTER_JAVA_OPTS` and
+`ALLUXIO_WORKER_JAVA_OPTS`.</td></tr>
+</table>
+
+For example, if you would like to setup a Alluxio master at `localhost` that talks to a HDFS also running at `localhost`, and enable Java
+remote debugging at port 7001, you can do so using:
+
+{% include Configuration-Settings/more-conf.md %}
+
+One can either set these variables through shell or set them in `conf/alluxio-env.sh`. If this file does not exist yet,
+you can create one from a template we provided in the source code using:
+
+{% include Common-Commands/copy-alluxio-env.md %}
+
+Alternatively, Alluxio can help users bootstrap the `conf/alluxio-env.sh` file by running
```

Comment:
yes, we should guide users towards using the bootstrap-conf script over copying the template manually